### PR TITLE
[23.2] make release titles consistent

### PR DESCRIPTION
**Pull request by martenson**
before
Screenshot 2024-01-22 at 3 06 54 PM

after
Screenshot 2024-01-22 at 5 12 16 PM

btw the paren typo is fixed

## License
- [x] I agree to license these and all my past contributions to the core galaxy codebase under the [MIT license](https://opensource.org/licenses/MIT).

### CODE REVIEW

**Review by jdavcs — APPROVED** [review]

**Comment by jdavcs** [comment]
I like it! Not merging yet so others can weigh in, in case there might alternative suggestions.
Thanks, @martenson!

**Comment by bernt-matthias** [comment]
Screenshot shows `))` for 23.2

Renaming only 23.0 would also make it consistent, or.

I would keep or remove the `v` in front of the version number.

**Comment by nsoranzo** [comment]
Would it make sense to do the same also for 22.05 and earlier releases?

**Comment by jdavcs** [comment]
@bernt-matthias I think he fixed the paren - he mentions that under the screenshot.

Renaming 23.0 would not fix the inconsistency caused by "Late". Look at the first word in each item: you have "Late", and then only names of months (except 23.0).

@nsoranzo I agree, it would make it more consistent.

**Comment by dannon** [comment]
I'm fine with whatever we choose here, but the intent was to switch to the more vague 'stable / mid / late' or 'early / mid / late' as the new convention.

**Comment by martenson** [comment]
@dannon I got that and agree that is a fine vocabulary for unpublished release. However the info when it actually went out is valuable and this makes it accessible.

File: doc/source/releases/24.0_announce.rst [hunk 1]
```diff
@@ -2,7 +2,7 @@
 :orphan:
 
 ===========================================================
-24.0 Galaxy Release
+24.0 Galaxy Release (Early 2024)
```
Comment (jdavcs) [inline doc/source/releases/24.0_announce.rst:5]: I think "24.0 Galaxy Release" would be sufficient. "Early" is vague (and I understand, it's by design). But, to reiterate, given that it doesn't adjust well when changing release frequency, as well as our (occasional) tendency to push back the release date weeks if not months, I'd keep it simple - 24.0. Besides, the "early" term feels almost like "early release", as in preliminary.

Comment (dannon) [reply doc/source/releases/24.0_announce.rst:5]: I like this as-is -- stating to expect the release "Early 2024", not that this nomenclature is a part of the release name.

**Review by dannon — APPROVED** [review]

**Pull request merged**